permissions/plugin-better-pipeline-flowgraph-table.yml

```
@@ -0,0 +1,9 @@
+---
+name: "better-pipeline-flowgraph-table"
+github: &GH "jenkinsci/better-pipeline-flowgraph-table-plugin"
+paths:
+- "io/jenkins/plugins/better-pipeline-flowgraph-table"
+developers:
+- "danielbeck"
+issues:
+  - github: *GH
```

To support https://github.com/jenkinsci/better-pipeline-flowgraph-table-plugin/pull/1:

```suggestion
  - github: *GH
cd:
  enabled: true
```